docs/tls/_index.md

```diff
@@ -6,55 +6,61 @@ weight: 500
+### CANamespaces
 
-    // TLSExpiryThreshold is the Vault TLS certificate expiration threshold in Go's Duration format.
-    // default: 168h
-    TLSExpiryThreshold string `json:"tlsExpiryThreshold,omitempty"`
+The list of namespaces where the generated CA certificate for Vault should be distributed. Use ["*"] for all namespaces.
 
-    // TLSAdditionalHosts is a list of additional hostnames or IP addresses to add to the SAN on the automatically generated TLS certificate.
-    // default:
-    TLSAdditionalHosts []string `json:"tlsAdditionalHosts,omitempty"`
+Default value: *[]*
 
-    // CANamespaces define a list of namespaces where the generated CA certificate for Vault should be distributed,
-    // use ["*"] for all namespaces.
-    // default:
-    CANamespaces []string `json:"caNamespaces,omitempty"`
-```
+### ExistingTLSSecretName
+
+The name of the secret that contains a TLS server certificate, key, and the corresponding CA certificate. The secret must be in the **kubernetes.io/tls type secret keys + ca.crt key** format. If the attribute is set, the operator uses the certificate already set in the secret, otherwise it generates a new one.
+
+The `ca.crt` key is mandatory, otherwise the Bank-Vaults components can't verify the Vault server certificate.
+
+Default value: *""*
+
+### TLSAdditionalHosts
+
+A list hostnames or IP addresses to add to the SAN on the automatically generated TLS certificate.
 
-The `ca.crt` key is mandatory in `existingTlsSecretName` otherwise the Bank-Vaults components can't verify the Vault server certificate.
+Default value: *[]*
 
-## Using the generated custom TLS certificate with vault-operator:
+### TLSExpiryThreshold
 
-Using existing secret, which contains the TLS certificate, define `existingTlsSecretName` in the Vault custom resource.
+The expiration threshold of the Vault TLS certificate in Go Duration format.
 
-### Generating custom certificates with CFSSL for Bank-Vaults
+Default value: *168h*
 
-If you don't wish to use the Helm or Operator generated certificates the easiest way to create a custom certificate for Bank-Vaults is [CFSSL](https://github.com/cloudflare/cfssl).
-This directory holds a set of custom CFSSL configurations which are prepared for the Helm release name `vault` in the `default` namespace. Of course, you can put any other certificates into the Secret below, this is just an example:
+## Using the generated custom TLS certificate with vault-operator
 
-1. Create a CA first:
+To use an existing secret which contains the TLS certificate, define [existingTlsSecretName](#existingtlssecretname) in the Vault custom resource.
+
+## Generate custom certificates with CFSSL
+
+If you don't want to use the certificates generated by Helm or the Bank-Vaults operator, the easiest way to create a custom certificate for Bank-Vaults is using [CFSSL](https://github.com/cloudflare/cfssl).
+
+<!-- FIXME which directory? -->
```

Comment:
This was moved somewhere else by another set of refactoring: https://github.com/banzaicloud/bank-vaults-docs/tree/master/docs/tls